src/llvm/lib_llvm_ext.cr

```diff
@@ -154,19 +154,18 @@ lib LibLLVMExt
                                           bundle : LibLLVMExt::OperandBundleDefRef,
                                           name : LibC::Char*) : LibLLVM::ValueRef
 
-  {% unless LibLLVM::IS_38 || LibLLVM::IS_39 %}
-    fun write_bitcode_with_summary_to_file = LLVMExtWriteBitcodeWithSummaryToFile(module : LibLLVM::ModuleRef, path : UInt8*) : Void
-  {% end %}
+  fun write_bitcode_with_summary_to_file = LLVMExtWriteBitcodeWithSummaryToFile(module : LibLLVM::ModuleRef, path : UInt8*) : Void
 
   fun normalize_target_triple = LLVMExtNormalizeTargetTriple(triple : Char*) : Char*
-  fun basic_block_name = LLVMExtBasicBlockName(basic_block : LibLLVM::BasicBlockRef) : Char*
   fun di_builder_get_or_create_array_subrange = LLVMExtDIBuilderGetOrCreateArraySubrange(builder : DIBuilder, lo : UInt64, count : UInt64) : LibLLVM::MetadataRef
 
   fun target_machine_enable_global_isel = LLVMExtTargetMachineEnableGlobalIsel(machine : LibLLVM::TargetMachineRef, enable : Bool)
   fun create_mc_jit_compiler_for_module = LLVMExtCreateMCJITCompilerForModule(jit : LibLLVM::ExecutionEngineRef*, m : LibLLVM::ModuleRef, options : LibLLVM::JITCompilerOptions*, options_length : UInt32, enable_global_isel : Bool, error : UInt8**) : Int32
 
-  {% unless LibLLVM::IS_38 %}
-    # LLVMCreateTypeAttribute is implemented in LLVM 13, but needed in 12
+  # LLVMCreateTypeAttribute is implemented in LLVM 13, but needed in 12
+  {% if LibLLVM::IS_LT_130 %}
     fun create_type_attribute = LLVMExtCreateTypeAttribute(ctx : LibLLVM::ContextRef, kind_id : LibC::UInt, ty : LibLLVM::TypeRef) : LibLLVM::AttributeRef
+  {% else %}
+    fun create_type_attribute = LLVMCreateTypeAttribute(ctx : LibLLVM::ContextRef, kind_id : LibC::UInt, ty : LibLLVM::TypeRef) : LibLLVM::AttributeRef
```

Comment:
This should go into `LibLLVM`, not here. Same for the `LLVMDIBuilder*` functions above

Reply:
But in LibLLVM it's only defined for LLVM > 13. So we'd have to switch at the callsite. IMO it's easier this way, even though it's a bit sloppy.

Reply:
And the DIBuilder functions still use `LibLLVMExt::DIBuilder` type, hence I opted to leave the bindings here where all the other DIBuilder functions are defined as well.